emdummy/__main__.py

```diff
@@ -26,13 +26,32 @@ def main():
 
     # The relevant part of config.py
     # from emdummy import EmDummy
-    em_dummy = ('emdummy', 'EmDummy', 'EXAMPLE (The friendly name of EmDummy used in REST API form)',
-                ('Params', 'goes', 'here'), {'source_fields': {'form'},  # Source field names
-                                             'target_fields': ['star']})  # Target field names
-    tools = [(em_dummy, ('dummy', 'dummy-tagger', 'emDummy'))]
+    em_dummy = (
+        'emdummy', # module name
+        'EmDummy', # class
+        'EmDummy, just add stars to `form`', # friendly name used in REST API form
+        (), # params (currently none)
```

Comment:
The word "params" is inaccurate as one must distinguish args from kwargs. In line 33 one can suply args and in line 34 a dictionary of kwargs (currently `source_fields` and `target_fields`).

Please correct the comment to include the above.